### DISCUSSION TITLE
Authenticate multiple wallets for a single user?

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1550] [Developer Support] [answered]
_This questions was originally posted on the WalletConnect Discord!_

Hey everyone! -- I have a question, with WalletConnect is it possible to connect + authenticate several different wallets for a single user? -- for example Metamask and coinbase, etc.

**Comment by xzilja** [comment, marked as answer]
At the moment, you'll have to disconnect and re-connect to switch wallets, but this is deff something we will be looking into after stable release as we start working on "quality of life" improvements 👍
